README.md

```diff
@@ -22,7 +22,7 @@ Please make sure you have these dependencies first before building.
 
 ```
 gtk4
-libadwaita-1
+libadwaita-1-0
```

Comment:
Why?

Reply:
Im not sure on this one @dusansimic , when I was installing and adding the pre-reqs on the Ubuntu dev host I had, libadwaita-1 returned no results, only when searching libadwaita-1-0 , should I revert?

Reply:
Sounds good no worries :)